mc-sema/cfgToLLVM/x86Instrs_MOV.cpp

```diff
@@ -573,6 +573,14 @@ static InstTransResult translate_MOVao (NativeModulePtr natM, BasicBlock *& bloc
     Function *F = block->getParent();
     Module *M = F->getParent();
 
+    // this is awful, but sometimes IDA detects the immediate
+    // as a memory reference. However, this instruction can only
+    // have an immediate, so this is safe
+    if( ip->has_imm_reference || ip->has_mem_reference ) {
```

Comment:
We sorta have the same issue on binja. We'll assign "PossibleAddressToken" to immediates that might be addresses. Adding our stack references cleaned up most of them, but some might exist. 

We are planning on implementing Alex's "confidence" suggestion at some point which may help as we'd be able to assign confidence based on the token type so that, for example, addresses identified through data flow as certainly pointers would be annotated as such, while others would have lower confidence. The confidence stuff is still just conceptual at this point though.